sdk/monitor/azure-monitor-opentelemetry-exporter/tests/test_base_exporter.py

```diff
@@ -215,22 +250,25 @@ def test_transmission_200(self):
         "APPLICATIONINSIGHTS_STATSBEAT_DISABLED_ALL": "false",
         }
     )
-    def test_statsbeat_200(self):
+    @mock.patch("azure.monitor.opentelemetry.exporter.statsbeat._statsbeat.collect_statsbeat_metrics")
+    def test_statsbeat_200(self, stats_mock):
```

Comment:
Not familiar with this style. Is stats_mock the mock path of collect_statsbeat_metrics? That automatically adds it as an argument?